src/Epic/canRender.ts

```diff
@@ -0,0 +1,26 @@
+import { BrazeMessageProps } from './index.stories';
```

Comment:
It seems _slightly_ odd to be importing this from a story file. I guess it won't end up in the prod bundle because it's a type, but I think I'd vote to move `BrazeMessageProps` to its own file and import it here (and in the story if we need it there too).